api/models/UserPassword.coffee

```diff
@@ -0,0 +1,63 @@
+# UserPassword.coffee
+#
+# @description :: Accepts and validates incoming POST request for user password.
+# @docs        :: http://sailsjs.org/documentation/concepts/models-and-orm/models
+
+module.exports =
+  # Allow only whitelisted attributes.
+  schema: true
+
+  # Turn off migrations.
+  migrate: 'safe'
+
+  # Turn off auto fields.
+  autoCreatedAt: off
+  autoUpdatedAt: off
+  autoPK: off
+
+  # Validation rules.
+  attributes:
+    user_id:
+      type: 'string'
+      required: yes
+      hexadecimal: yes
+      defaultsTo: ""
+    email:
+      type: 'string'
+      required: yes
+      email: yes
+      defaultsTo: ""
+    mobile:
+      type: 'string'
+      required: yes
+      defaultsTo: ""
+    application_id:
+      type: 'string'
+    email_template:
+      type: 'string'
+
+  # Validate presence at least one of fields.
```

Comment:
`# Validate presence at least one of fields.` should be `# Validate the presences of at least one of the fields.`